SU-SWS/caw_profile
# CAW21-106 Created side by side sweetener list with filter on the left

# READY FOR REVIEW

# Summary
- Created side by side list of sweeteners with the filter on the lft.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. in the stack root directory run `composer require su-sws/caw_profile:dev-CAW21-106 --no-update; composer update -n`
2. `blt drupal:sync --site=cardinalatwork`
3. create a basic page, and add a list paragraph to a One Column layout. Choose "Sweeteners" and "Side by Side" display for the list paragraph
4. Set the page layout as "Full Width"
5. View the page in all break points and ensure it looks ok.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
